sdcm/utils/decorators.py

```diff
@@ -188,7 +188,7 @@ def wrapped(*args, **kwargs):  # pylint: disable=too-many-branches, too-many-loc
                 return res
             hdr_stats = []
             hdr_latency = CSHdrHistogram(args[0].loaders.logdir)
-            for timestamp in range(int(start - 300), int(end + 300), 300):
+            for timestamp in range(int(start + 60), int(end + 60), 600):
```

Comment:
It was start - X, end + X, 300

now

it is  start + Y, end +Y,  Z

Is this intentionally?

Reply:
fixed